src/Pos/Generator/Block/Param.hs

```diff
@@ -39,16 +42,46 @@ instance Buildable AllSecrets where
             (length _asSecretKeys)
             (HM.keys _asSecretKeys)
 
+-- | Parameters for transactions payload generation.
+data TxGenParams = TxGenParams
```

Comment:
Because transaction payload generation is `txp` thing. But I see there is a TODO about moving tx payload generator to txp and I think it's ok to postpone it.